sensorlib/src/util/CarlaUtils.py

```diff
@@ -105,14 +120,14 @@ def get_semantic_tag_name(tag_id):
         """
         return CarlaUtils.CarlaCityObjectLabelLookup.get(tag_id, "NONE")
 
-    @staticmethod
-    def get_semantic_tag_id(tag_name):
-        """
-        Get the integer semantic tag ID for a given tag name.
-        :param tag_name: The string tag name to look up.
-        :return: The tag ID.
-        """
-        return carla.CityObjectLabel.names.get(tag_name, 0)
+    # @staticmethod
```

Comment:
Why are we commenting this out. Please leave a TODO comment with reasoning for this.